### DISCUSSION TITLE
How to change the data directory?

### DISCUSSION THREAD

**Discussion by mths1** [vernemq/vernemq#2187] [Q&A]
Hello all

short question: What is the "official" way for chaning the data directory (without beeing overwritten by cuttlefish)? I'd like to move the data directory to some different storage, without a soft-link...

**Comment by ioolkos** [comment]
In short, it's decided before building a release. There's a `platform_data_dir` setting in respective config files. The schema files then can reference the `{{platform_data_dir}}` and create subdirectories for application use.
